### DISCUSSION TITLE
Resizing, Moving eframe window

### DISCUSSION THREAD

**Discussion by BestClaws** [emilk/egui#570] [Q&A]
Hello, I'd like to dynamically resize the eframe's native window, and also change its position. I can't find anything in the docs to do this. Do i have to use something like egui_glium? any help?

**Comment by MrTanoshii** [comment]
Resizing: https://docs.rs/eframe/0.18.0/eframe/struct.Frame.html#method.set_window_size
Position: https://docs.rs/eframe/0.18.0/eframe/struct.Frame.html#method.set_window_pos


Edit: Found `set_window_pos`
Edit2: Proper referencing `eframe::Frame`
